joomla/joomla-cms
# 4.4-dev

If it still isn't right, please let me know.
That's the only way I learn

Pull Request for Issue # .

This is new to me.
If I did it wrong please let me know.

Summary of Changes
I changed fixed translation to translatable strings

Testing Instructions
Actual result BEFORE applying this Pull Request
In localAdapter.php there was untranslatable text

Expected result AFTER applying this Pull Request
In localAdapter.php the text is now translatable using string
### Link to documentations
Please select:
- [ X] No documentation changes for docs.joomla.org needed

- [ X] No documentation changes for manual.joomla.org needed
